.editorconfig

```diff
@@ -14,15 +14,6 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 charset = utf-8
 
-# Python files
-[*.py]
-indent_size = 4
-# isort plugin configuration
-known_first_party = invenio_s3
```

Comment:
did you remove this to rely on the sorting done by `black`?

Reply:
yes. on the pages of black and isort it says that profile=black is enough for compatibility. further the configuration in .editorconfig may be in conflict with the black/isort formatting. therefore it is better to remove it.

the sorting is done with isort, but with the black profile they play nicely with each other with the formatting of black

https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html?highlight=isort#profile